servers/xr/xr_vrs.cpp

```diff
@@ -104,8 +104,13 @@ void XRVRS::set_vrs_render_region(const Rect2i &p_vrs_render_region) {
 RID XRVRS::make_vrs_texture(const Size2 &p_target_size, const PackedVector2Array &p_eye_foci) {
 	ERR_FAIL_COND_V(p_eye_foci.is_empty(), RID());
 
+#ifdef RD_ENABLED
 	int32_t texel_width = RD::get_singleton()->limit_get(RD::LIMIT_VRS_TEXEL_WIDTH);
 	int32_t texel_height = RD::get_singleton()->limit_get(RD::LIMIT_VRS_TEXEL_HEIGHT);
+#else
+	int32_t texel_width = 0;
+	int32_t texel_height = 0;
```

Comment:
I don't know if VRS can be used on OpenGL but this probably warrants a sane default either way, CC @BastiaanOlij @dsnopek.

Reply:
VRS cannot be used with OpenGL. I think this whole function should probably just be skipped if we don't have `RenderingDevice`, and just return `RID()`